java_src/foam/nanos/auth/UserAndGroupAuthService.java

```diff
@@ -195,7 +171,7 @@ public Boolean validateUser(User user) throws IllegalStateException {
    * Just return a null user for now. Not sure how to handle the cleanup
    * of the current context
    */
-  public X logout(X x) {
-    return this.getX().put("user", null);
+  public void logout(X x) {
+    this.getX().put("user", null);
```

Comment:
Contexts are immutable, so this line doesn't do anything. Which is good, cause it isn't needed.